implementation/BUILD

```diff
@@ -1,4 +1,5 @@
 load("@vsomeip//bazel:cmake_like_configure_file.bzl", "cmake_like_configure_file")
+load("@vsomeip//bazel:version.bzl", _vsomeip_version_str = "vsomeip_version_str")
 
 cmake_like_configure_file(
```

Comment:
There's already a version define in this configured header. My assumption would be that the version information should come from here right @rosds?